### ISSUE TITLE
Add Python 3.5 syntax to Python in langs.model.xml

### ISSUE THREAD

**Issue by AraHaan** [notepad-plus-plus/notepad-plus-plus#2640]
### Description of the Issue
Notepad++ does not know about the Python 3.5+ syntax of ``async def`` and ``await``

### Steps to Reproduce the Issue
1. Get a Python 3.5+ Script.
2. Open it with Notepad++
3. await and async are not highlighted when they should be.

### Expected Behavior
``async`` and ``await`` getting highlighted

### Actual Behavior
No highlighting for ``async`` and ``await`` whatsoever.

### Debug Information

```
Notepad++ v7.2.2   (32-bit)
Build time : Nov 27 2016 - 16:31:07
Path : E:\Program Files (x86)\Notepad++\notepad++.exe
Admin mode : ON
Local Conf mode : OFF
OS : Windows 7
Plugins : MenuIcons.dll mimeTools.dll NppConverter.dll nppcrypt.dll NppExec.dll NppExport.dll NppFTP.dll NPPJSONViewer.dll NppSaveAsAdmin.dll PluginManager.dll PyNPP.dll Python Indent.dll PythonScript.dll ReloadButton.dll Translate.dll ZenCoding-Python.dll zoomdisabler.dll 
```

![197799fcb12b0a176e763e45852e4228 1](https://cloud.githubusercontent.com/assets/15173749/20863099/f85afae4-b985-11e6-9bee-482bd178c007.png)

### Solution Change ``python`` entry in langs.model.xml to:

```xml
        <Language name="python" ext="py pyw" commentLine="#">
            <Keywords name="instre1">and as assert break class continue def del elif else except exec False finally for from global if import in is lambda None not or pass print raise return True try while with yield async await</Keywords>
        </Language>
```
And some other changes to actually make it highlight.

**Comment by dail8859** [comment]
> And some other changes to actually make it highlight.

Other than the keywords, what are you referring to?

You can create a PR with the new keywords.

**Comment by AraHaan** [comment]
ok I will, what file should I edit on the repo to add them in?

**Comment by dail8859** [comment]
The one you already mentioned.

**Comment by dail8859** [comment]
Closed by 855e581e44a7535888554f08eb2939b23b31f384
